app/validation/validators.py

```diff
@@ -71,11 +72,35 @@ def __init__(self, minimum=None, maximum=None, messages=None):
 
     def __call__(self, form, field):
         data = field.data
+        error_message = None
         if data is not None:
             if self.minimum is not None and data < self.minimum:
-                raise validators.ValidationError(self.messages['NEGATIVE_INTEGER'])
+                error_message = self.messages.get('OUT_OF_RANGE', self.messages['NUMBER_TOO_SMALL'])
             elif self.maximum is not None and data > self.maximum:
-                raise validators.ValidationError(self.messages['INTEGER_TOO_LARGE'])
+                error_message = self.messages.get('OUT_OF_RANGE', self.messages['NUMBER_TOO_LARGE'])
```

Comment:
It all depends how we want to handle the error messages for these. This way allows us to use the new style message only where needed. Since OUT_OF_RANGE is the new message not currently used I have made the assumption that if we add this to a schema then it is because we want to use it. Otherwise we will use the currently used too large/small messages.